OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java

```diff
@@ -67,9 +67,11 @@ public static File getSelectedFile(String extension, String description, DialogT
         int returnVal = 0;
         switch (type) {
             case SAVE:
+                chooser.setDialogTitle("Save As");
```

Comment:
What I meant is why we have separated it in this method, inside the switch loop. We can merge the cases like
```
case SAVE:
case SAVEAS:
    <code>
```